### ISSUE TITLE
Create "Subsection" status in Coda

### ISSUE THREAD

**Issue by LeMurphant** [StampyAI/stampy-ui#560]
Not directly about the website, but we want to create a "Subsection" status in coda to put subsections in. We in Copy can move them once the status exists.

https://discord.com/channels/677546901339504640/1209956074984181830/1214998246083133510

Steven and I think we should move the actual Google Docs to "Live on site", @plexish thoughts on this?

**Comment by plexish** [comment]
No strong opinion from me

**Comment by bryceerobertson** [comment]
@LeMurphant is this still an issue?

**Comment by LeMurphant** [comment]
@bryceerobertson yes it is still relevant, the coda status of these articles is still kind of in a weird limbo. IIRC @mruwnik said he would do it.

**Comment by mruwnik** [comment]
I'm not sure I understand how this should work. My current understanding would be something like there's a "Subsection" Coda status - anything with this status should be ignored by the parser (as it shouldn't have text) or whatever, but it certainly shouldn't display anything in the frontend?

**Comment by LeMurphant** [comment]
@mruwnik that is correct. There's no visible functionality change in the frontend, we just want to make it clear that these articles are not "not started". You can add the "don't display the text" explicitly, but it should be necessary since it's working ok right now.
![not started](https://github.com/user-attachments/assets/ac7ac29c-a5db-418e-b9f8-94680b53d9d2)

**Comment by mruwnik** [comment]
the frontend should already support it, so you can try it out with a couple of subsection articles

**Comment by LeMurphant** [comment]
@mruwnik I tried changing all the sections of "intro to AI safety" to subsections, and after a few hours the navigation for the whole site was broken. Seems related although I'm confused how it stopped it from loading the "advanced" sections.

**Comment by LeMurphant** [comment]
Fixed now, seems to be working

**Comment by mruwnik** [comment]
should be fixed now

**Comment by LeMurphant** [comment]
Confirmed to work, thanks.
